### DISCUSSION TITLE
Getting Started with Meshtastic® Kit | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#2004] [Q&A]
# Getting Started with Meshtastic® Kit | Seeed Studio Wiki

Get Started with Wio Tracker Dev Kit for Meshtastic®

https://wiki.seeedstudio.com/meshtastic_kit_wio_tracker_1110/

**Comment by eerke2021** [comment]
Which Pin is the GPS connected to?

**Reply by eerke2021** [reply]
Using the Grove GPS, not the onboard gos connector.

**Reply by Jessie219-web** [reply]
Grove GPS is connected to the UART port
RXD: P0.24
TXD: P0.25
3V3
GND

**Comment by eerke2021** [comment]
Thanks. I’ve updated the settings in Meshtastic. On 18 Dec 2024, at 02:07, Jessie ***@***.***> wrote:﻿
Grove GPS is connected to the UART port
RXD: P0.24
TXD: P0.25
3V3
GND

**Comment by StrangePelaton** [comment]
I only get NaN for both temp and humidity. Is there something besides telemetry that I need to set up?

**Reply by Jessie219-web** [reply]
did you enable the sensor? navigate to Meshtastic App: `Settings`-> `Telemetry(Sensors)` -> `Enable` sensors.

**Comment by StrangePelaton** [comment]
Yes, everything was enabled. It shows telemetry coming from the Wio Tracker 1110 Dev Kit in the node list when i am connected to my other node, meaning the telemetry is working, but what it transmits is simply "NaN" for all fields. I used the official meshtastic web flasher correct firmware for the board, and have tried several recent and current versions of the app. I get telemetry with accurate numbers from every other node except the Wio Tracker 1110 Dev Kit.
    On Monday, June 23, 2025 at 05:16:12 AM EDT, Jessie ***@***.***> wrote:  
 
 


did you enable the sensor? navigate to Meshtastic App: Settings-> Telemetry(Sensors) -> Enable sensors.
